tests/let/timportc.nim

```diff
@@ -0,0 +1,24 @@
+discard """
+targets: "c cpp js"
+"""
+
+when defined(c) or defined(cpp):
+  {.emit:"""
+  const int TEST1 = 123;
+  #define TEST2 321
+  """.}
+
+when defined(js):
+  {.emit:"""
+  const TEST1 = 123;
+  const TEST2 = 321; // JS doesn't have macros, so we just duplicate
+  """.}
+
+let
+  TEST0 = 1
```

Comment:
nit: this would be a styleCheck violation; maybe turn everything to lowercase or perhaps for the macro at least `let test2 {.importc: "TEST2", nodecl.}: cint

Reply:
Hmm, I was just trying to emulate what this kind of code often looks like. C oftes uses UPPERCASE for constants and macros and I think many of the C auto-wrappers does the same thing as C (as long as it's possible).